### PR TITLE
Docs: Beef up the FAQ

### PR DESCRIPTION
**Pull request by sideshowbarker**
This is an attempt at beefing up the FAQ a bit by adding some detailed-yet-succinct overview stuff about the project and its history.

https://github.com/sideshowbarker/ladybird/blob/overview.md-add/Documentation/FAQ.md#what-are-some-things-that-ladybird-isnt (and forward from there) has the rendered (HTML) view of the additions to the FAQ.

### CODE REVIEW

**Comment by trflynn89** [comment]
I don't have much of an opinion on having the document, but I think including so many current stats is just asking for stale docs. I wouldn't include "Project activity metrics", "Current code size", and test scores. This also duplicates some information in the README, like how to build.

**Comment by sideshowbarker** [comment]
> I think including so many current stats is just asking for stale docs. I wouldn't include "Project activity metrics", "Current code size", and test scores.

OK, thanks — I pushed an update with those either completely cut, or else reduced down to just links or relative numbers.

> This also duplicates some information in the README, like how to build.

Completely cut some stuff that was duplicative.

**Comment by circl-lastname** [comment]
Why have basically a second FAQ? Could these be consolidated

**Comment by sideshowbarker** [comment]
> I feel this could be to some extent consolidated with the FAQ.

Yeah, I don’t disagree. So I’ve now pushed a separate commit putting it there instead.

That does have the advantage of giving us one single _“here’s the general (but detailed) **project** info for all interested parties (including non-contributors/media/etc)”_ thing to point people to.

File: Documentation/FAQ.md [hunk 1]
```diff
@@ -42,3 +42,91 @@ Here’s a very short summary: Work on what eventually became Ladybird started o
...
 
 LibHTML eventually became [LibWeb](https://github.com/LadybirdBrowser/ladybird/tree/master/Userland/Libraries/LibWeb) — which in turn eventually grew into being the core part of the browser engine and browser to which, on 4 July 2022, [the name _Ladybird_ was given](https://www.youtube.com/watch?v=X38MTKHt3_I&t=29s).
+
+## What are some things that Ladybird _isn’t_?
+
+- Not a Blink/Chromium shell.
+- Not a WebKit port.
+- Not a Firefox fork.
+
+## What makes Ladybird/[Ladybird Browser Initiative](https://ladybird.org/) different?
```
Comment (ADKaster) [inline Documentation/FAQ.md:52]: now that this is in FAQ, this bullet has common threads with `What does 'Independent' mean, if you're including third party dependencies?`

File: Documentation/FAQ.md [hunk 2]
```diff
@@ -42,3 +42,91 @@ Here’s a very short summary: Work on what eventually became Ladybird started o
...
+
+- Fully independent: Written from scratch, using no code from any other browser engine.
+- Singular focus: Doing only one single thing: building a new browser engine and browser.
+- No monetization: Will never take funding from default search deals or any other forms of user monetization, ever.
+
+## Is there a short history/timeline about how Ladybird got started?
```
Comment (ADKaster) [inline Documentation/FAQ.md:58]: This bullet has common threads with `Where did Ladybird come from?`

File: Documentation/FAQ.md [hunk 3]
```diff
@@ -42,3 +42,91 @@ Here’s a very short summary: Work on what eventually became Ladybird started o
...
+- 2024 July: [Ladybird Browser Initiative](https://ladybird.org/posts/announcement/) launched by Andreas and GitHub co-founder [defunkt](https://twitter.com/defunkt) (Chris Wanstrath).
+
+## Are there video/audio announcements and interviews about the start of the Ladybird Browser Initiative?
+
+- Ladybird Browser Initiative [announcement video](https://www.youtube.com/watch?v=k9edTqPMX_k) from defunkt explaining the project _raison dʼêtre_ + goals (July 2024).
+- [Why we need Ladybird](https://changelog.com/podcast/604#t=5:08): _Changelog_ podcast interview with Andreas and defunkt (August 2024); [transcript](https://changelog.com/podcast/604#transcript); [chapters](https://changelog.com/podcast/604#chapters).
+- [Eron Wolf announcement grant of $200K](https://youtu.be/p6k9qcRpW_k) from [FUTO](https://www.futo.org/about/what-is-futo/) to the project (August 2024).
+- [Eron Wolf interview with Andreas](https://youtu.be/4xhaAAcKLtI) (August 2024).
+
+## Is there a release roadmap?
```
Comment (ADKaster) [inline Documentation/FAQ.md:73]: perhaps this slots in nicely after `## When will you implement `$THING`?`

**Review by ADKaster — APPROVED** [review]

**Pull request merged**